src/components/molecules/Visualizer/Engine/Cesium/Box/utils.ts

```diff
@@ -0,0 +1,152 @@
+import {
+  Axis,
+  Cartesian2,
+  Cartesian3,
+  Cartographic,
+  HeadingPitchRoll,
+  Matrix3,
+  Matrix4,
+  Quaternion,
+  Ray,
+  Scene,
+  Transforms,
+  TranslationRotationScale,
+} from "cesium";
+
+import { Property } from ".";
+
+const translationWithClamping = (
```

Comment:
You have this in the Tileset file, too. Since this declaration is in a utils file, can you just export this one and use it in Tileset too?
It might make sense to move it in to the Cesium utils to be accessed by both

Reply:
Sure. I moved this method to utils. [refactor: move translationWithClamping to Cesium/utils](https://github.com/reearth/reearth-web/pull/357/commits/166258bc1c306d90ef8a4998ed2f7131a85c1bfd)